modes/longitudinal/src/toolbarButtons.js

```diff
@@ -189,14 +189,17 @@ export default [
       type: 'primary',
     },
   },
-  // TODO: Toggle
   {
     id: 'Cine',
-    type: 'ohif.action',
+    type: 'ohif.toggle',
+    config: {
+      groupName: 'primaryTool',
```

Comment:
I think we're using this to identify which tools in a group only allow for the selection of a single tool within the group. Toggling CINE is more of a separate on/off the shows/hides UI controls.

Reply:
> I think we're using this to identify which tools in a group only allow for the selection of a single tool within the group. Toggling CINE is more of a separate on/off the shows/hides UI controls.

I see. Maybe the name is misleading then? I thought that what you described would be a radio or something